common/tests/interlocked_hl_win32_ut/interlocked_hl_win32_ut.c

```diff
@@ -0,0 +1,843 @@
+// Copyright(C) Microsoft Corporation.All rights reserved.
```

Comment:
I think you just want this in the win32 folder

Reply:
I don't understand what you mean by "in the win32 folder".

---
In reply to: [437082398](https://github.com/Azure/azure-c-pal/pull/1#discussion_r437082398) [](ancestors = 437082398)